docs/source/checks/lda.rst

```diff
@@ -0,0 +1,237 @@
+   which check you want to run: in this case
+   :class:`~shift_detector.checks.lda_check.LdaCheck`.
+3. Finally, you print the result with :meth:`~shift_detector.detector.Detector.evaluate`.
+
+Result
+++++++
+
+::
+
+    LDA Check
+    Examined Columns: ['Entry']
+    Shifted Columns: ['Entry']
+
+    Column 'Entry has a diff in topic 1 of ':
+        0.01
+    Column 'Entry has a diff in topic 2 of ':
+        -0.015
+    Column 'Entry has a diff in topic 4 of ':
+        0.012
+    Column 'Entry has a diff in topic 9 of ':
+        0.019
+    Column 'Entry has a diff in topic 11 of ':
+        0.021
+    Column 'Entry has a diff in topic 13 of ':
+        0.016
+    Column 'Entry has a diff in topic 16 of ':
+        0.02
+    Column 'Entry has a diff in topic 17 of ':
+        -0.019
+    Column 'Entry has a diff in topic 18 of ':
+        -0.014
+    Column 'Entry has a diff in topic 19 of ':
+        -0.033
+    Column 'Entry has a diff in topic 20 of ':
+        -0.015
+
+.. image:: ../images/lda_barplots.png
+  :width: 1200
+   .. image:: ../images/wordclouds.png
+  :width: 1200
+ .. image:: ../images/pyldavis.png
+  :width: 1200
+
+Interpretation
+++++++++++++++
+
+The above result can be interpreted as follows:
+
+1.  First, a listing of all columns considered by the check is displayed.
+2.  Then all columns that are detected as shifted are displayed with their
+    according relative difference as value.
+3.  After that a 3 plots are displayed for every shifted column.
+4.  The first shows the relative frequency of each topic for both datasets
+    for the same column the plot is titled as.
+5.  The second shows a word cloud of the most frequent words per topic for
+    the column.
+6.  The third shows an interactive visualization that shows all topics of the
+    column as clusters on a two-dimensional grid. Each cluster is clickable
+    and on click it shows on right next to the clusters a bar chart of the
+    30 most relevant terms for the selected topic. Per default the relevance
+    metric lambda is set to 1. This means that the terms are sorted descending
+    by their term frequency inside the topic.
+    However in the paper [Sievert2014]_ that introduced this relevance metric,
+    it is advised to set the relevance slider to around 0.6 to make the terms
+    the most descriptive of the topic for a human reader.
+
+
+Parameters
+----------
+
+:ref:`LdaCheck` provides the following parameters:
+
+``significance``:
+    This parameter expects a float between 0 and 1 and impacts if a column is
+    detected as shifted or not.
+    The lower you choose ``significance`` the higher is the probability that
+    the difference in the topic distributions of the two datasets exceeds the siginficance.
+
+    The default value is 0.1. This means that the percentage of documents of any topic
+    from the first dataset has to differ from the second dataset at least 10%.
```

Comment:
Perhaps a different name is better to avoid confusion with the significance of the statistical tests